meteor-app/imports/ui/member/add/member-add-control.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import PropTypes from 'prop-types';
+import { Button, Icon } from 'semantic-ui-react'
+
+const MemberAddControl = (props) => {
+  return (
+    <div>
+      {
+        (props.step > 1) &&
+        <Button type='button' floated='left' onClick={props.backStep}>
+          <Icon name='arrow left' />
+          Back
+        </Button>
+      }
+      {
+        (props.step < 3) &&
```

Comment:
This component is kind of generic, but you have built into it a knowledge of their being exactly 3 steps. If you add a step, then you need to modify this component too. Better to pass in a `stepCount` prop to it.